app/celery/broadcast_message_tasks.py

```diff
@@ -16,13 +16,15 @@ def send_broadcast_event(broadcast_event_id):
         current_app.logger.info(f'CBC Proxy disabled, not sending broadcast_event {broadcast_event_id}')
         return
 
+    broadcast_event = dao_get_broadcast_event_by_id(broadcast_event_id)
     for provider in current_app.config['ENABLED_CBCS']:
-        # TODO: Decide whether to send to each provider based on platform admin, service level settings, broadcast
-        # level settings, etc.
-        send_broadcast_provider_message.apply_async(
-            kwargs={'broadcast_event_id': broadcast_event_id, 'provider': provider},
-            queue=QueueNames.NOTIFY
-        )
+        if broadcast_event.service.allowed_broadcast_provider in {None, provider}:
```

Comment:
For my own learning, what made you pick a set here over a list or a tuple?

Reply:
it won't make a difference here, but in a larger list of things it's computationally quicker since set lookup is O(1) (memory location is defined by a hash of the contents, so looking up is a constant time complexity) while tuples behave like a list, so lookup is O(n) (as it has to scan the entire list).

obviously for two items (especially when it's almost always going to be None), the difference in this is minimal, but i think it's a good habit to use sets more for this kind of stuff